tests/runtime/openquantum/test_openquantum_runtime.py

```
@@ -247,6 +249,46 @@ def test_device_submit_batch(mock_requests_put, provider):
+        "status_code": 403,
+        "message": [
+            "Please visit https://www.openquantum.com to accept the Terms of Use "
+            "before submitting jobs."
+        ],
+        "type": "TERMS_OF_USE_REQUIRED",
+        "error_code": "req-1",
+    }
+    response.text = ""
+    response.reason = "Forbidden"
+
+    with pytest.raises(QbraidRuntimeError, match="openquantum.com"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)